shared/common-adapters/back-button.desktop.js

```diff
@@ -18,11 +18,12 @@ class BackButton extends Component<Props> {
     return (
       <div style={collapseStyles([styles.container, this.props.style])} onClick={e => this.onClick(e)}>
         <Icon type="iconfont-arrow-left" style={styles.icon} color={this.props.iconColor} />
-        {this.props.title !== null && (
-          <Text type="BodyPrimaryLink" style={this.props.textStyle} onClick={e => this.onClick(e)}>
-            {this.props.title || 'Back'}
-          </Text>
-        )}
+        {this.props.title !== null &&
```

Comment:
why not just pass title = null?

Reply:
Ah, because the page has a title -- I'm using `<HeaderHocHeader title={..}`>, and that component passes the same title it receives down to `<BackButton>` here, so passing a null title isn't an option given that I want the header title.